spec/grid_spec.rb

```diff
@@ -0,0 +1,225 @@
+require_relative "../tic-tac-toe/lib/grid.rb"
+require_relative "spec_helper.rb"
+
+RSpec.describe Grid do
+  subject { Grid.new }
+
+  describe '#new' do
+    it 'works' do
+      result = Grid.new
+      expect(result).not_to be_nil
+    end
+  end
+  describe '#put_piece' do
+    it 'works' do
+      grid = Grid.new
+      dbl=double("Put_piece", :index=>3, :symb =>'x')
+      result=grid.put_piece(dbl.index, dbl.symb)
+      expect(result).not_to be_nil
+    end
```

Comment:
Nothing here is showing that the piece is into the Grid even if is not returning nil
What I purpose is to check whether or not the board at that position, contains the value :X